presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java

```diff
@@ -116,7 +120,10 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
                     toTimeStamp(queryStats.getCreateTime()),
                     toTimeStamp(queryStats.getExecutionStartTime()),
                     toTimeStamp(queryStats.getLastHeartbeat()),
-                    toTimeStamp(queryStats.getEndTime()));
+                    toTimeStamp(queryStats.getEndTime()),
+
+                    Optional.ofNullable(queryInfo.getErrorType()).map(Enum::name).orElse(null),
+                    Optional.ofNullable(queryInfo.getErrorCode()).map(ErrorCode::getCode).orElse(null));
```

Comment:
i think `ErrorCode::getName` would be more useful

(i'd leave the column name as is)